model/src/cfd.rs

```diff
@@ -1913,6 +1917,44 @@ impl From<AdaptorSignature> for secp256k1_zkp::EcdsaAdaptorSignature {
 
 impl_sqlx_type_display_from_str!(AdaptorSignature);
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct Transaction(bitcoin::Transaction);
```

Comment:
if anything, the old way of  serialising  was not that hard, it just needed the 
`        #[serde(with = "hex_transaction")] Transaction` 🙃

Reply:
Really? 🙈

Reply:
ah, but then I'd need to use serde for serializing it which will give me a json right?

Reply:
Boy, how did I survive without knowing this ...

Reply:
downside is that it serializes it to a json string, i.e. including `""`. I'll leave it as is if this is fine?

Reply:
I think you can overcome that with using `Hyphenated`